frontend/src/components/BackupViewDialog.vue

```diff
@@ -22,6 +25,23 @@ const backup = ref<Backup | undefined>(undefined);
 const backupForEval = ref<CreateRequest | undefined>(undefined);
 const jobItems = ref<{ job: Job; restore: RestoreResponse | undefined }[]>([]);
 
+const cleanupTrashcan = () => {
+  if (backup.value?.id) {
+    DefaultService.postTrashcansCleanUp(backup.value?.id)
```

Comment:
Maybe a action dialog would be nice, before performing the action, to prevent accidental deletion